tests/test_restart.py

```diff
@@ -84,3 +85,31 @@ def test_restart(sim_executed):
     # check if params_simul.xml is updated
     params_in_filesystem = load_params(sim_executed.path_run)
     assert params.output.session_id == params_in_filesystem.output.session_id
+
+
+def test_phys_fields_uninit(sim):
+    """Should error if trying to load / get_var without executing init_reader."""
+    with pytest.raises(RuntimeError):
+        sim.output.phys_fields.load()
+
+    with pytest.raises(RuntimeError):
+        sim.output.phys_fields.get_var()
+
+
+@pytest.mark.slow
+def test_phys_fields(sim_executed):
+    sim_executed.output.phys_fields.init_reader()
+
+    try:
+        ds = sim_executed.output.phys_fields.load()
+        ux = sim_executed.output.phys_fields.get_var("ux")
+    except ValueError:
+        pytest.xfail(
+            reason=(
+                "Pymech does not support non-box meshes, and snek5000-phill has "
```

Comment:
We could use snek5000-cbox for another test!

Reply:
I can try.

Reply:
I will push a test with cbox. But it has to be updated.

```py
__________________ ERROR at setup of test_phys_fields_get_var_before_load ___________________

    @pytest.fixture(scope="module")
    def sim_cbox_executed():
        from snek5000_cbox.solver import Simul
    
>       params = Simul.create_default_params()

tests/conftest.py:144: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../snek5000-cbox/src/snek5000_cbox/solver.py:66: in create_default_params
    params.output._set_child(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <snek5000.params.Parameters object at 0x7ff5d3e16b50>

<output HAS_TO_SAVE="True" session_id="0" sub_directory="">
  <...="['pymech', 'pymech_stats']" reader="pymech"/>  

  <history_points coords="None" write_interval="100"/>  

</output>

tag = 'phys_fields', attribs = {'write_interval_pert_field': 1000}

    def _set_child(self, tag, attribs=None):
        """Add a child (of the same class) to the container."""
        if tag in self.__dict__:
>           raise ValueError(f'The tag "{tag}" is already used.')
E           ValueError: The tag "phys_fields" is already used.

../../../.pyenv/versions/3.8.9/envs/snek/lib/python3.8/site-packages/fluiddyn/util/paramcontainer.py:351: ValueError
```

Reply:
Ah I think this is because I merge something that needs #95. The simpler would be to merge #95 and to rebase this PR on main.